src/server/views/dashboard/lists.njk

```diff
@@ -17,32 +19,32 @@
 
     {% include "./partials/dashboard-boxes.njk" %}
 
-    <table class="govuk-table govuk-!-margin-top-4 dashboard-table">
+    <table class="govuk-table dashboard-table">
+
       <thead class="govuk-table__head">
-        <tr class="govuk-table__row">
-          <th scope="col" class="govuk-table__header govuk-table__header--dashboard govuk-!-width-one-fifth">Type</th>
-          <th scope="col" class="govuk-table__header govuk-table__header--dashboard">Country</th>
-          <th scope="col" class="govuk-table__header govuk-table__header--dashboard">Live</th>
-          <th scope="col" class="govuk-table__header govuk-table__header--dashboard">Action needed</th>
-          <th scope="col" class="govuk-table__header govuk-table__header--dashboard govuk-!-width-one-sixth">Last review</th>
-          <th scope="col" class="govuk-table__header govuk-table__header--dashboard govuk-!-width-one-sixth">Next review</th>
-          <th scope="col" class="govuk-table__header govuk-table__header--dashboard">Admins</th>
-          <th scope="col" class="govuk-table__header govuk-table__header--dashboard govuk-table__header--numeric"></th>
-        </tr>
+      <caption class="govuk-visually-hidden">Your lists</caption>
+        {% for cell in tableHeaders %}
+            {{ tableHeaderCell(cell) }}
+        {% endfor %}
+        {{ tableHeaderCell({name: "listId"}) }}
+
       </thead>
       <tbody class="govuk-table__body">
         {% for list in lists %}
           <tr class="govuk-table__row">
-            <td scope="row" class="govuk-table__header">
-                <a href="{{ dashboardRoutes.listsItems.replace(':listId', list.listId) }}" class="govuk-link">{{ _.startCase(list.type) }}</a>
+            <td scope="row" class="govuk-table__header  dashboard-table__cell">
+                <a href="{{ dashboardRoutes.listsItems.replace(':listId', list.listId) }}" class="govuk-link">
+                  {{ "Translators and Interpreters" if list.type === 'translatorsInterpreters' else _.startCase(list.type) }}
+                  <span class="govuk-visually-hidden">in {{ list.country }}</span>
```

Comment:
This makes a lot of sense. Should we have mode text for if there's no last annual review start date and next annual review date so that the user can tell the difference between the two, or do you think None and No review is enough?